projects/zlib-ng/build.sh

```diff
@@ -25,5 +25,15 @@ make -j$(nproc) clean
 make -j$(nproc) all
 make -j$(nproc) check
 
-find . -name 'compress_fuzzer' -exec cp -v '{}' $OUT ';'
+# First remove the existing *_fuzzers and *_corpus.zip from $OUT.
+find $OUT -name '*_fuzzer' -exec rm -f '{}' ';'
```

Comment:
this is redundant, we build the image from scratch, there shouldn't be any leftovers

Reply:
I see that I can add --clean to remove the artifacts.